### PR TITLE
Turks and caicos set wikipedia as membership source

### PR DESCRIPTION
**Pull request by ondenman**
Part of: https://github.com/everypolitician/everypolitician-data/issues/25274

This is a step towards adding the 2016 term.

The wikipedia source provides party, area and term information. As such, wikipedia should be the membership source.

(The next step will be to remove the `area` and `term` columns from the elected person source.)

Notes:
Incoming member Edwin Astwood is not listed in the official source but is listed in the Wikipedia source. Changing the Wikipedia source type to membership means he now gets imported.

Two existing memberships have changed. Constituency names in the Wikipedia article use an ampersand in place of 'and'. Amanda Missick's constituency has changed from Cheshire Hall and Richmond Hill to Cheshire Hall **&** Richmond Hill. Ricardo Don Hue Gardiner's constituency changes from North and Middle Caicos to North **&** Middle Caicos.

```
Add memberships from sources/archive/official-2012-appointed-members.csv
Add memberships from sources/morph/wikipedia.csv
Merging with sources/archive/official-2012-elected-members.csv
Merging with sources/morph/wikidata.csv
Adding GenderBalance results from sources/gender-balance/results.csv
  ⚥ data for 12; 12 added


Top identifiers:
  6 x wikidata
  1 x freebase

Creating names.csv
Persons matched to Wikidata: 6 ✓ | 13 ✘
  No wikidata: Clarence Selver (b036609e-30da-4c49-82bd-219d51f7ac2c)
  No wikidata: Edwin Astwood (115262f1-45e6-4a17-ae36-86c31bcdd57d)
  No wikidata: Amanda A. Misick (0bab1627-2702-4bd3-a4e5-a4e9a262fa90)
  No wikidata: Vaden Delroy Williams (ed3f718c-b43a-4f6a-9558-6b329aae764d)
  No wikidata: Porsha Monique Stubbs Smith (51931afb-361d-4d2a-9873-c692d6e79b78)
  No wikidata: Ruth Delores Blackman (f6d8ace4-f66c-4260-a906-b17938a4dfdf)
  No wikidata: Sean Rickard Astwood (0c52f20d-9d0b-45c9-82cc-e95b9c1dec1c)
  No wikidata: John Philips (3e8718af-2588-44ef-9516-551a19be3f13)
  No wikidata: Goldray McMillin Ewing (99a7157d-5b4c-46d7-a194-67f5e80da4fa)
  No wikidata: George Alexander Lightbourne (dc1c0d1b-bcd4-483a-8be0-9bd8754e35f4)
Parties matched to Wikidata: 2 ✓ | 1 ✘
  No wikidata: unknown (party/unknown)
Areas matched to Wikidata: 0 ✓ | 11 ✘
```

### CODE REVIEW

**Comment by ondenman** [comment]
screen shot 2017-03-01 at 10 28 34

**Comment by tmtmtmtm** [comment]
> Incoming member Edwin Astwood is not listed in the official source but is listed in the Wikipedia source. Changing the Wikipedia source type to membership means he now gets imported

Do you know _why_ he's not listed on the official source?

**Comment by ondenman** [comment]
@tmtmtmtm I cannot find a reason why the member is not listed on the official site.

The site was redesigned somewhen between 3rd February and [23rd March](http://web.archive.org/web/20150323214026/http://www.gov.tc/) 2015. Before the redesign, it did not list assembly members. Edwin Astwood is not listed on the redesigned site.

However, [this](http://tcweeklynews.com/year-in-review-p6777-127.htm) page reports that Edwin Astwood was active in July 2015. Also, [this](http://www.sppdtci.com/election-years) site, which appears to belong to the T&C Office of Statistics, lists the member in the 2012 election results.

**Comment by tmtmtmtm** [comment]
@ondenman what's the thinking behind 53cdbaf770b538482e75cbdb5a12961c5e5b1854? There's no explanation in the commit message, and it makes the eventual diffs much harder to follow…

**Comment by ondenman** [comment]
@tmtmtmtm I moved the wikipedia source up as I was under the assumption that membership sources had to go before person sources. Is that not right?

**Comment by tmtmtmtm** [comment]
@ondenman nope.

**Comment by ondenman** [comment]
@tmtmtmtm I've removed 53cdbaf770b538482e75cbdb5a12961c5e5b1854 commit and rebased over master.

File: data/Turks_and_Caicos_Islands/Assembly/sources/instructions.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+ {
```
Comment (tmtmtmtm) [inline data/Turks_and_Caicos_Islands/Assembly/sources/instructions.json]: this seems like an unexpected change…

**Comment by ondenman** [comment]
@tmtmtmtm I've re-ordered the commits and force pushed.

**Comment by ondenman** [comment]
@tmtmtmtm I've removed the spurious space from the instructions file. The idmap file is removed in: 739c6f65491d53f694a179e3c71bae7be88026c6 -- unless I've misunderstood.

**Comment by tmtmtmtm** [comment]
> The idmap file is removed in: 739c6f65491d53f694a179e3c71bae7be88026c6 -- unless I've misunderstood

Ah, this is well disguised on the https://github.com/everypolitician/everypolitician-data/pull/26967/files view, as it's being treated as a move to `sources/reconciliation/official-2012-elected-members.csv`, rather than a delete. Apologies for missing that, though as previously discussed it's a good idea to explicitly respond to each review point, whether with a pointer to the commit addressing it (as you've done with the JSON formatting here), to ask more about it, or to explain why you don't think it's valid.

**Pull request merged**